### PR TITLE
Doc: Update image rendering with HTTPS configuration

### PR DESCRIPTION
**Pull request by AgnesToulet**
**What is this feature?**
Add documentation on the image renderer about HTTPS configuration

**Special notes for your reviewer:**
See https://github.com/grafana/grafana-image-renderer/pull/527 for more context

### CODE REVIEW

**Review by juanicabanas — APPROVED** [review]
LGTM!

**Review by chri2547 — APPROVED** [review]
LGTM! Thanks for the contribution.

**Comment by diegommm** [comment]
Hi! Greetings from the Release Guild. Just letting you know I'm updating your PR's milestone to [11.2.x](https://github.com/grafana/grafana/milestone/529). If you need to see your changes in a previous version, please consider adding a backport label.

Thank you!

File: docs/sources/setup-grafana/image-rendering/_index.md [hunk 1]
```diff
@@ -207,6 +207,47 @@ HTTP_PORT=0
 }
 ```
 
+#### HTTP protocol
+
+{{% admonition type="note" %}}
+HTTPS protocol is supported from v3.11.0 and later.
```
Comment (chri2547) [inline docs/sources/setup-grafana/image-rendering/_index.md:213]:
```suggestion
HTTPS protocol is supported in the image renderer v3.11.0 and later.
```

**Pull request merged**